gcc/doc/extend.texi

```diff
@@ -21118,6 +21118,59 @@ Generates the @code{cv.macsRN} machine instruction.
 Generates the @code{cv.machhsRN} machine instruction.
 @end deftypefn
 
+These built-in functions are available for the CORE-V bit manipulation machine
+architecture. For more information on CORE-V built-ins, please see
+@uref{https://github.com/openhwgroup/core-v-sw/blob/master/specifications/corev-builtin-spec.md#pulp-bit-manipulation-builtins-for-32-bit-cores}
+
+@deftypefn {Built-in Function} {int32_t} __builtin_riscv_cv_bitmanip_extract (int32_t, uint16_t)
+Generated assembler @code{cv.extract} is the uint16_t operand is a constant. Generated assembler
```

Comment:
is -> if (also in all succeeding blocks)

Reply:
Updated